robotiq_action_server/src/robotiq_c_model_action_server.cpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * ActionServer interface to the control_msgs/GripperCommand action
+ * for a Robotiq C-Model device
+ */
+
+#include "robotiq_action_server/robotiq_c_model_action_server.h"
+
+// To keep the fully qualified names managable
+namespace ras = robotiq_action_server;
```

Comment:
This namespace aliasing should not be required.  The only namespace in any file should be the package namespace.